tools/jenkins/run_performance.sh

```diff
@@ -34,4 +34,4 @@ set -ex
 # Enter the gRPC repo root
 cd $(dirname $0)/../..
 
-tools/run_tests/run_performance_tests.py -l c++ node ruby csharp python --netperf --category smoketest
+tools/run_tests/run_microbenchmark.py -c --diff_perf origin/master
```

Comment:
Instead of `origin/master`, `origin/$ghprbTargetBranch` can be used so that the diff is calculated against the branch the PR wants to merge into.